docs/_data/metrics/social_media_2/summary.yml

```diff
@@ -0,0 +1,34 @@
+title: Leeds 2023 twitter Summary Statistics
+service:
+  twitter:
+    metrics:
+      audience_last: 16510
+      engagements_total: 44394
+      audience_gained_total: 1432
+      impressions_total: 2140846
+    filter-range: 2022-01-01:2022-12-31
```

Comment:
Change `filter-range` and `filter-col` to `filter_range` and `filter_col`, wherever they appear